src/device-registry/validators/kya.validators.js

```
@@ -0,0 +1,1537 @@
+        return ObjectId(value);
+      }),
+  ],
+
+  notEmptyBody: [
+    body()
+      .notEmpty()
+      .custom((value) => {
+        return !isEmpty(value);
+      })
+      .withMessage("The request body should not be empty."),
+  ],
```

Biome (1.9.4)

[error] This property value named notEmptyBody is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named notEmptyBody


(lint/suspicious/noDuplicateObjectKeys)